modules/message-lane/src/lib.rs

```diff
@@ -286,7 +320,7 @@ decl_module! {
 		#[weight = 0] // TODO: update me (https://github.com/paritytech/parity-bridges-common/issues/78)
 		pub fn receive_messages_delivery_proof(origin, proof: MessagesDeliveryProofOf<T, I>) -> DispatchResult {
 			let _ = ensure_signed(origin)?;
```

Comment:
We might consider incentivising this separately. Currently it's assumed that the relayer who delivered the messages is interested to get rewards. However, we should make sure that other relayers are not going to get blocked by him (which currently is the case).
We might split the total fee between the relayers who delivered a message and the submitter of the delivery proof. CCing #316 here.

Reply:
> We might consider incentivising this separately. Currently it's assumed that the relayer who delivered the messages is interested to get rewards. However, we should make sure that other relayers are not going to get blocked by him (which currently is the case).

Other relayers always have an opportunity to unblock themselves. But I agree that incentivisation hook may be done here. Actually, probably adding `confirmation_transaction_submitter` to `pay_relayer_reward()` would be enough.